CHANGES.rst

```diff
@@ -2,10 +2,18 @@
 CHANGES
 =======
 
-3.7.2 (unreleased)
+4.0.0 (unreleased)
 ------------------
 
-- Nothing changed yet.
+- Add support for Python 3.4, 3.5, 3.6 and PyPy.
+
+- Fix the ``evolve`` view to use independent transactions instead of
+  committing or aborting the thread-local current transaction.
```

Comment:
I admit that's (probably) a bit of hypothetical future-proofing, and the tests don't actually exercise that case (nor do they fail with this case). I can roll it back if that seems best.